components/body/index.jade

```diff
@@ -15,16 +15,20 @@ section#collecting
   .body-section-content__section-two.js-section-content
     .columns.left-gutter-formatter
       .column
-        include ./mixins/artist-container.jade
-        - options = {path: './images/body-2-1.jpg', artistName: 'Claude Monet', artwork: 'Le Parlement, soleil couchant (The House of Parliament, at Sunset),', artworkDate: '1900-1901,', location: 'Christie\'s', className: 'small-rectangle'}
-        +artist-container(options)
-        - options = {path: './images/body-2-3.jpg', artistName: 'Jeff Koons', artwork: 'Metallic Venus,', artworkDate: '2010-2012,', location: 'Gagosian Gallery', className: 'vertical-rectangle'}
-        +artist-container(options)
+        a.artwork(href="https://www.artsy.net/artwork/claude-monet-le-parlement-soleil-couchant-the-houses-of-parliament-at-sunset")
```

Comment:
There's probably an opportunity here to fold this link into your mixin, you could just pass the artwork slug (i.e. `claude-monet-le-parlement-soleil-couchant-the-houses-of-parliament-at-sunset`) since that is the only unique thing about these links.